metabase/metabase
# add connection impersonation permissions UI

Epic https://github.com/metabase/metabase/issues/30020

### Description

This PR adds UI for the connection impersonation settings modal.

### How to verify

1. Run Metabase EE
2. Go to admin -> people
3. Edit any user details and add at least one attribute
4. Go to admin -> permissions
5. On any database change the permission to "impersonated"
6. Ensure it shows the role–attribute mapping modal

### Demo

<img width="604" alt="Screen Shot 2023-04-14 at 9 03 36 PM" src="https://user-images.githubusercontent.com/14301985/232172870-17673d70-6a52-4060-a07e-4725b4d83ba7.png">

<img width="1055" alt="Screen Shot 2023-04-14 at 9 03 27 PM" src="https://user-images.githubusercontent.com/14301985/232172947-42d305a8-c7d6-4373-8f14-cacfba66a080.png">

### Checklist

- [-] Tests have been added/updated to cover changes in this PR — not much to test here, will add specs in further PRs


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/30132)
